### PR TITLE
Add dnsexit support (based on #52)

### PR DESCRIPTION
**Pull request by rrthomas**

### CODE REVIEW

**Comment by rrthomas** [comment]
I have updated the patch from #52 to master, and merged the fix from @truesalo.

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Dynamic DNS services currently supported include:
     ClouDNS     - See https://www.cloudns.net
     dinahosting - See https://dinahosting.com
     Gandi       - See https://gandi.net
+	dnsexit		- See https://dnsexit.com/ for details
```
Comment (SuperSandro2000) [inline README.md:35]: Please use spaces.

File: ddclient.in [hunk 2]
```diff
@@ -3408,7 +3423,119 @@ sub nic_dyndns2_update {
...
+sub nic_dnsexit_update {
+    debug("\nnic_dnsexit_update -------------------");
+
+    my %status = (
+         '0'        => [ 'good',    'Success' ],
+         '1'        => [ 'nochg',   'IP is the same as the IP on the system' ],
+         '2'        => [ 'badauth', 'Invalid password' ],
+         '3'        => [ 'badauth', 'User not found' ],
+         '4'        => [ 'nochg',   'IP not changed. To save our system resourc
+es, please don\'t post updates unless the IP got changed.' ],
```
Comment (SuperSandro2000) [inline ddclient.in:3478]:
```suggestion
         '4'        => [ 'nochg',   'IP not changed. To save our system resources, please don\'t post updates unless the IP got changed.' ],
```

File: README.md (hunk 1 above)
Comment (rrthomas) [reply README.md]: Fixed, thanks!

File: ddclient.in (hunk 2 above)
Comment (rrthomas) [reply ddclient.in]: Fixed, thanks!

File: ddclient.in [hunk 3]
```diff
@@ -3408,7 +3423,119 @@ sub nic_dyndns2_update {
...
+
+    ## update each configured host
+    foreach my $h (@_) {
+        my $ip = delete $config{$h}{'wantip'};
+        info("setting IP address to %s for %s", $ip, $h);
+        verbose("UPDATE:","updating %s", $h);
+
+        # Set the URL that we're going to update
+        my $url;
+        $url  = "http://$config{$h}{'server'}$config{$h}{'script'}";
```
[the remark this answers is not in the document]
Comment (rrthomas) [reply ddclient.in]: Luckily it seems that they do now support https! Fixed.

**Comment by DaveSophoServices** [comment]
@rrthomas  Please check the develop branch, that dnsexit works well.

**Comment by rrthomas** [comment]
@DaveSophoServices thanks, will do that by the end of this weekend.

**Comment by rrthomas** [comment]
@DaveSophoServices it's working for me, thanks!

**Pull request merged**